app/services/PsrSubmissionService.scala

```diff
@@ -48,17 +48,19 @@ class PsrSubmissionService @Inject()(
   moneyBorrowedTransformer: MoneyBorrowedTransformer,
   sharesTransformer: SharesTransformer,
   bondTransactionsTransformer: BondTransactionsTransformer,
-  otherAssetTransactionsTransformer: OtherAssetTransactionsTransformer
+  otherAssetTransactionsTransformer: OtherAssetTransactionsTransformer,
+  declarationTransformer: DeclarationTransformer
 ) {
 
-  def submitPsrDetails(
+  def submitPsrDetailsWithUA(
```

Comment:
This renaming was required because submitPsrDetails had been overloaded already. In base method, isSubmitted added as a new parameter with default value causes ambiguity with renamed method.